functions/routes/authentication.js

```diff
@@ -4,7 +4,7 @@ var router=express.Router();
 // a route which we will be using to authenticate API calls for NIN pending our approval, it requires a post request to be activated
 router.post("/",(request,response)=>{
```

Comment:
name this endpoint appropriately 
`router.post("/example",(request,response)=>{`